hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -370,6 +370,9 @@ module top_${top["name"]} #(
   assign rv_core_ibex_irq_timer = intr_rv_timer_timer_expired_hart0_timer0;
   assign rv_core_ibex_hart_id = '0;
 
+  // Unconditionally enable the late debug feature
```

Comment:
```suggestion
  // Unconditionally disable the late debug feature
```
I admit that this _disabling late debug_ input isn't very intuitive :-) What this change does (as intended) is to _disable_ late debug, hence "enabling early debug", which is what you want.

Reply:
Ok, I thought it was to enable it until a a late stage :D